quiche/src/lib.rs

```diff
@@ -9849,8 +9852,9 @@ mod tests {
             pipe.server.stream_send(4, b"hello", false),
             Err(Error::Done)
         );
+
         assert_eq!(
-            pipe.server.stream_send(8, b"hello", false),
+            pipe.server.stream_send(9, b"hello", false),
```

Comment:
I suspect the use of stream 8 was intentional. The server can't write on a client-initiated stream that hasn't been opened yet and the assertion tested that. Later assertions demonstrate that the server stream writes succeed after the client opened the stream.

Reply:
Makes sense. I changed them to now just expect InvalidStreamState.